string cmd
string poseName
---
string message